### ISSUE TITLE
Entry point not found for report

### ISSUE THREAD

**Issue by lbeltrame** [robinandeer/chanjo-report#9]
Upon installing chanjo_report, I get

````
ImportError: Entry point ('chanjo.subcommands', 'report') not found
````

As far as I can tell, both chanjo and chanjo-report have been installed correctly.
Removing chanjo-report gets rid of the error, but I actually need the functionality. ;)

**Comment by robinandeer** [comment]
I will investigate this error. Any more details on the commands you run to reproduce the error would be very helpful :smiley:

**Comment by robinandeer** [comment]
The reason was that I was loading the subcommands as if they were all defined inside the ``chanjo`` package. This broke the CLI when installing external subcommands. Now fixed in v2.3.2!

Thanks for the bug report!
